### PR TITLE
Add page for whatis

### PR DESCRIPTION
**Pull request by sbrl**

### CODE REVIEW

**Comment by sbrl** [comment]
Great suggestions, @waldyrious. I've updated it.

File: pages/linux/whatis.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# whatis
+
+> Display one-line manual page descriptions.
```
Comment (waldyrious) [inline pages/linux/whatis.md]: How about "one line descriptions from manual pages"?

**Comment by waldyrious** [comment]
Sorry @sbrl, I didn't mean to ask for the hyphen in "one-page" to be removed. That was a mishap on my side 😅

**Comment by sbrl** [comment]
😃

File: pages/linux/whatis.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`whatis {{command}}`
+
+- Don't cut the description off at the end of the line:
+
+`whatis --long {{command}}`
+
+- Search man page descriptions with a regular expression:
+
+`whatis --regex {{regular expression}}`
```
Comment (sbrl) [inline pages/linux/whatis.md]: Done.

**Comment by waldyrious** [comment]
@sbrl commit 3204c76 seems to have undone the fix from commit c49671c (the period in the main description). Please fix that, and we should be ready to merge :)

**Comment by sbrl** [comment]
@waldyrious Oops! I've fixed that.

**Comment by waldyrious** [comment]
@agnivade do the honors :)

**Comment by agnivade** [comment]
Here we go ! :+1:

**Comment by sbrl** [comment]
Thanks!

**Pull request merged**